### DISCUSSION TITLE
How do I use C-Sharp classes and code that I put in a file in a system folder in the virtual file system? Not an error.

### DISCUSSION THREAD

**Discussion by FunnyDoggs** [CosmosOS/Cosmos#2815] [General]
I haven't created an operating system yet, but I would like to know how to do this before I start.
#### Area of Cosmos - What area of Cosmos are we dealing with?
File system

#### Version - Were you using the User Kit or Dev Kit? And what User Kit version or Dev Kit commit (Cosmos, IL2CPU, X#)?
I would like to know how to do it in the User Kit.

**Comment by FunnyDoggs** [comment]
I want to know how to do this because a lot of tutorials say that it's possible to store code on the file system, but they never explain how to use the classes in them.

**Comment by zarlo** [comment]
right now there is no way to compile C# in cosmos but you can run basic .net programs with https://github.com/MishaProductions/DotNetParser

**Reply by FunnyDoggs** [reply]
Do you mean putting it in your operating system to run it? How would it work in an operating system?

**Reply by zarlo** [reply]
you can see a way to do it here https://github.com/aura-systems/Aura-Operating-System/tree/feature/dotNetParser/SRC
